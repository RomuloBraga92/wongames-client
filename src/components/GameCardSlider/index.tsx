import * as S from './styles'

const GameCardSlider = () => (
  <S.Container>
    <h1>GameCardSlider</h1>
  </S.Container>
)

export default GameCardSlider
