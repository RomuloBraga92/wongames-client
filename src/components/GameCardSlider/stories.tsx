import { Story, Meta } from '@storybook/react/types-6-0'
import GameCardSlider from './'

export default {
  title: 'GameCardSlider',
  component: GameCardSlider,
} as Meta

export const Default: Story = () => <GameCardSlider />
