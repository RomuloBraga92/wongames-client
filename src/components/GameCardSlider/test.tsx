import { render, screen } from '@testing-library/react'

import GameCardSlider from './'

describe('<GameCardSlider />', () => {
  it('should render the heading', () => {
    const { container } = render(<GameCardSlider />)

    expect(
      screen.getByRole('heading', { name: /GameCardSlider/i }),
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
