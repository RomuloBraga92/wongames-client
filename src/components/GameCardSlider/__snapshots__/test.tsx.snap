// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GameCardSlider /> should render the heading 1`] = `
<main
  class=""
>
  <h1>
    GameCardSlider
  </h1>
</main>
`
